mate_menu/plugins/applications.py

```diff
@@ -1534,15 +1557,18 @@ def updateBoxes( self, menu_has_changed ):
                     else:
                         launcherNames.append(launcherName)
         except Exception as e:
-            print e
+            print(e)
```

Comment:
I think this small change can be moved to https://github.com/ubuntu-mate/mate-menu/pull/36, for consistency :slightly_smiling_face:

Reply:
Ok, now that it's removed from here, it should be added to #36 for sure...